cove_360/templates/cove_360/explore.html

```diff
@@ -219,9 +238,9 @@ <h4>Other</h4>
     <div class="panel-heading" data-toggle="collapse" data-target="#quality-accuracy-body">
       <h3 class="panel-title panel-title-explore">
         <span class="font-tick question" aria-hidden="true"></span>
-          {% trans "Quality / Accuracy" %}
```

Comment:
I thought we were going to leave `accuracy` in the check name.

Reply:
@KDuerden 's feedback was that she wanted a single-word title, and I think I agree. I've also kept the internal naming of quality-accuracy as we might want to change that in the future!